easybuild/tools/options.py

```diff
@@ -1444,16 +1445,7 @@ def det_location(opt, prefix=''):
         pretty_print_opts(opts_dict)
 
 
-def parse_options(args=None, with_include=True):
-    """wrapper function for option parsing"""
-    if os.environ.get('DEBUG_EASYBUILD_OPTIONS', '0').lower() in ('1', 'true', 'yes', 'y'):
-        # very early debug, to debug the generaloption itself
-        fancylogger.logToScreen(enable=True)
-        fancylogger.setLogLevel('DEBUG')
-
-    if args is None:
-        args = sys.argv[1:]
-
+def unroll_arguments(args):
     # unroll arguments that correspond to a combo of single-letter options
     # this is done to avoid interpreting -rD like "--robot D" instead of "--robot -D"
```

Comment:
comments should be change to a proper docstring (and we should add a test for this new function)

Reply:
This function was inlined again, so no description (or seperate test) needed anymore